server/meta.jsx

```diff
@@ -140,6 +140,7 @@ const ALLOWED_ATTRS = [
     SDK_SETTINGS.CLIENT_TOKEN,
     SDK_SETTINGS.MERCHANT_ID,
     SDK_SETTINGS.PARTNER_ATTRIBUTION_ID,
+    SDK_SETTINGS.DATA_POPUPS_DISABLED,
```

Comment:
I removed the "DATA_" part of this name in sdk-constants since it was the only setting using it.
```suggestion
    SDK_SETTINGS.POPUPS_DISABLED,
```